src/databases/CMakeLists.txt

```diff
@@ -379,6 +382,7 @@ if(VISIT_PARADIS)
 endif()
 
 CHECK_THIRDPARTY_SATISFIED(H5Part H5PART)
+CHECK_THIRDPARTY_SATISFIED(WData WData)
```

Comment:
Our CHECK_THIRDPARTY_SATISFIED macro is for checking if third-party (external) dependencies have been met. It looks to me like you included WData source with the plugin and that there are no external dependencies. Have I read that correctly?  If there are no external dependencies, this line can be removed and your plugin added to the REQUIRED_DATABASE_PLUGINS list lower down in this file.

Reply:
This is something I was not sure, as some of the databases uses third-party names as their names. This was edited manually. To be clear, we do not use any other dependencies

Reply:
I have included the CMakeLists.txt as it is. File is generated on our local server. After the commit pipeline tests failed. I suppose that its due to this part:

> CMAKE_MINIMUM_REQUIRED(VERSION 3.8 FATAL_ERROR)
> SET(VISIT_PLUGIN_DIR "/home/makowski/.visit/3.1.2/linux-x86_64/plugins")
> INCLUDE("/usr/local/visit-3.1.2/3.1.2/linux-x86_64/include/PluginVsInstall.cmake")
> INCLUDE("/usr/local/visit-3.1.2/3.1.2/linux-x86_64/include/VisItLibraryDependencies.cmake")

How should I follow?